app/config/config.php

```diff
@@ -90,3 +90,59 @@
         'cacheDir' => $projectDir . 'web/cache',
     ],
 ]);
+
+// Session
+// SESSION_HANDLER - redis
+// SESSION_PATH - tcp://127.0.0.1:6379
+if (($sessionHandler = getenv('SESSION_HANDLER')) && ($sessionPath = getenv('SESSION_PATH'))) {
+    $config['session'] = [
+        'save_handler' => $sessionHandler,
+        'save_path' => $sessionPath,
+    ];
+}
+
+// Backend Session
+// BACKEND_SESSION_HANDLER - redis
+// BACKEND_SESSION_PATH - tcp://127.0.0.1:6379
+if (($backendSessionHandler = getenv('BACKEND_SESSION_HANDLER')) && ($backendSessionPath = getenv('BACKEND_SESSION_PATH'))) {
+    $config['backendsession'] = [
+        'save_handler' => $sessionHandler,
+        'save_path' => $sessionPath,
+    ];
+}
+
+// Model Cache
+// MODEL_CACHE_HANDLER - redis
+// MODEL_CACHE_HOST - tcp://127.0.0.1:6379/1
+if (($modelCacheHandler = getenv('MODEL_CACHE_HANDLER')) && ($modelCacheHost = parse_url(getenv('MODEL_CACHE_HOST')))) {
+    $config['model'] = [
+        'redisHost' => $modelCacheHost['host'],
+        'redisPort' => $modelCacheHost['port'],
+        'redisDbIndex' => (int)$modelCacheHost['path'],
+        'cacheProvider' => $modelCacheHandler
+    ];
+}
+
+// HTTP Cache
+// HTTP_CACHE_HANDLER - redis
+// HTTP_CACHE_HOST_0 - tcp://127.0.0.1:6379/1
+if ($httpCacheHandler = getenv('HTTP_CACHE_HANDLER')) {
```

Comment:
thats zend cache and not http cache :wink:

Reply:
Yeah I know.. Already changed that :)